### ISSUE TITLE
Image rending in front of all layers

### ISSUE THREAD

**Issue by clayrisser** [tamagui/tamagui#1259]
### Current Behavior

Image is rending on top of all layers when trying to stack it behind a layer using a ZStack.

### Expected Behavior

The image should render according the natural stack of layers. Why is there a zIndex in the image?

```ts
const StyledImage = styled(RNImage, {
  name: 'Image',
  position: 'relative',
  source: { uri: '' },
  zIndex: 1, // I don't think this should be here
})
```

### Tamagui Version

```markdown
1.30.23
```

### Reproduction

```markdown
https://github.com/tamagui/tamagui/blob/master/packages/image/src/Image.tsx#L23
```

**Comment by natew** [comment]
Fix in cc56f476ccc32c7c3fe4421144713a5c67c69eb1, will be released shortly.

**Comment by natew** [comment]
https://github.com/tamagui/tamagui/releases
